developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -175,20 +173,23 @@ Completed generating configuration:
 Aptos is running, press ctrl-c to exit
 ```
 
-This output contains information required for starting the Aptos CLI tool:
+Use the [Aptos CLI tool](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md) to interact with your local testnet. The above output contains information you will use for starting the [Aptos CLI tool](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md):
+
 * `Aptos root key path`: The root key (also known as the mint or faucet key) controls the account that can mint tokens. Available in the docker compose folder under `aptos_root_key`.
 * `Waypoint`: A verifiable checkpoint of the blockchain (available in the docker compose folder under waypoint.txt)
 * `REST endpoint`: The endpoint for the REST service, e.g., `http://127.0.0.1:8080`.
-* `ChainId`: The chain id uniquely distinguishes this network from other blockchain networks.
+* `ChainId`: The chain ID uniquely distinguishes this network from other blockchain networks.
 
-## Next Steps
+## Next steps
 
 At this point, you will have a special root account at `0x1` that can perform the mint operation. Follow up with:
 
 * [Your first transaction](/tutorials/your-first-transaction) to learn how to submit transactions.
 * [Your first Move module](/tutorials/your-first-move-module) to learn how to create Move modules.
 * [Interacting with the Aptos Blockchain](/transactions/interacting-with-the-aptos-blockchain) to learn how to mint coins.
 
-It is important to note that this guide does not include creating a faucet. That is left as an exercise for the reader.
+:::info
 
-[docker](https://docs.docker.com/get-docker/)
+This guide does not describe how to create a faucet.
+
```

Comment:
I am muddled on this statement. Both the methods actually start the faucet service, isn't it? Is this statement saying something different?